src/common/Utilities/Util.h

```diff
@@ -21,6 +21,7 @@
 
 #include "Define.h"
 #include "Errors.h"
+#include "Common.h"
```

Comment:
Util.cpp already include Common.h , please move the function implementation in the cpp